pkgs/development/python-modules/class-registry/default.nix

```diff
@@ -1,32 +1,32 @@
-{
-  buildPythonPackage,
-  fetchPypi,
-  lib,
-  nose,
-  six,
-  typing ? null,
-  isPy27,
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, nose
+, pythonOlder
 }:
 
 buildPythonPackage rec {
```

Comment:
This looks good but the only dependent package is now failing. If you are adding the v3 for your own use you may need to fork the package into v2 and v3. Otherwise please fix backblaze-b2.

Reply:
A python package is duplicated. We just need to remove the duplicated package and then it should just work.